### DISCUSSION TITLE
(optionally) use center of object to determine zone entering

### DISCUSSION THREAD

**Discussion by R5fan** [blakeblackshear/frigate#1219] [Ideas]
In my situation, the street before my house is at a ~45 degree angle in the camera view:
![image](https://user-images.githubusercontent.com/1439530/121820800-fefaaf80-cc94-11eb-921e-b16379678737.png)

Objects however, get a square box, .. at ~45 degree compared to the street. That means the bottom of the box, particularly if its a large car/truck, crosses very far downwards.  This isnt the best example, but it illustrates the problem:

![image](https://user-images.githubusercontent.com/1439530/121820789-f4d8b100-cc94-11eb-917c-282fe59f9451.png)

Now imagine its a long truck with a trailer and you can imagine the bottom of the box coming halfway my driveway. 
So  I would like the option to consider the center of the box to determine which zone its in.

**Comment by highaltidude** [comment]
From what I understand, the box is downsampled into a 300x300 image to pass through the ai. Is the longer truck not getting identified?

**Reply by R5fan** [reply]
The problem is not identification, its determining what zone it is in. Frigate uses the bottom center of the bounding box to determine if its in a zone., and given the 45 degree between the bounding box and the street, this leads to less than ideal results, with cars and trucks "entering" my drive way.
